packages/uui-color-swatch/lib/uui-color-swatch.element.ts

```diff
@@ -206,6 +252,16 @@ export class UUIColorSwatchElement extends LabelMixin(
     return this.color?.isLight() ?? false;
   }
 
+  @query('strong')
```

Comment:
If I'm not mistaken its unnecessary complex to only add the title if its overflowing.
I think we should always add the title, Both to keep it simpler and for the purpose of accessibility. :-)

Reply:
BTW. This comment is regarding the query and the `_detectOverflow ` method.

Reply:
I might have to mention it. I think we can just put the label on the button, similar to aria-label. And always have it there. So I think aria-label on the button can just be renamed to label.

Reply:
Im not entirely sure i get the last part - but I removed the overflow detection and added the title on the button now

Reply:
perfect, that was it :-)